pkg/api/types.go

```diff
@@ -754,6 +754,8 @@ type EndpointsList struct {
 type NodeSpec struct {
 	// Capacity represents the available resources of a node
 	Capacity ResourceList `json:"capacity,omitempty"`
+	// Cidr represents the IP range assigned to the node
+	Cidr string `json:"cidr,omitempty"`
```

Comment:
Sure, I am happy to change it to "CIDR". Also, I was thinking about "IPRange" or "PodIPRange" etc., so I am hoping somebody makes a final call how this should actually be named.